manifests/overlays/prod/statefulsets/argocd-application-controller_patch.yaml

```diff
@@ -1,14 +1,13 @@
 ---
 apiVersion: apps/v1
-kind: Deployment
+kind: StatefulSet
 metadata:
   name: argocd-application-controller
 spec:
   template:
     spec:
       containers:
       - name: argocd-application-controller
-        image: quay.io/aicoe/argocd:v1.7.4-3
```

Comment:
dont we have to mention a new image here?

Reply:
yeah, these are now added from the base which is pulled remotely from here: 
https://github.com/operate-first/continuous-deployment/blob/master/manifests/base/kustomization.yaml#L13

Reply:
The image is actually specified in upstream manifests now.

Reply:
ah gotchu